### DISCUSSION TITLE
rttm files?

### DISCUSSION THREAD

**Discussion by yfchau-news** [deepgram/community#985] [General help] [answered]
Just wonder if the API can return rttm data for diarization? Thank you.

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.

**Comment by deepgram-community** [comment, marked as answer]
No we currently don't support returning RTTM (Rich Transcription Time Marked) data for diarization.

I'm curious what is your use case for this file type?

> ![avatar](https://avatars.slack-edge.com/2023-02-20/4855338233600_fad9174023ee3f640ce1_32.png) This message was sent by John Vajda from Deepgram, via our community automation.

**Reply by yfchau-news** [reply]
Thank you for your reply. I am trying to evaluation diarization performance using rttms files.
